dotnet/src/Planners/Planners.Handlebars/Handlebars/HandlebarsPlannerOptions.cs

```diff
@@ -24,6 +24,16 @@ public sealed class HandlebarsPlannerOptions : PlannerOptions
     /// </remarks>
     public Func<Task<string>>? GetAdditionalPromptContext { get; set; }
 
+    /// <summary>
+    /// Callback that returns an override for the CreatePlan prompt.
+    /// </summary>
+    /// <remarks>
+    /// The callback should return a valid Handlebars template string. If this is set, the planner will use this prompt instead of the default prompt.
+    /// Devs can select any partial defined in Planners.Handlebars.CreatePlanPromptPartials namespace when constructing their own prompt.
+    /// No partials are included by default; make sure to select partials such as "{{> UserGoal }}" or {{> AdditionalContext}} if needed.
+    /// </remarks>
```

Comment:
Can you add a kernel syntax sample showing how to use this and add a link in the API documentation.

Reply:
Added `RunOverrideCreatePlanPromptSampleAsync` to Example65_HandlebarsPlanner.cs

Reply:
Seems like all of our API documentation is outdated, will work with @matthewbolanos to update docs for Handlebars Planner